CONTRIBUTORS.md

```diff
@@ -13,4 +13,6 @@ Edouard Oyallon
 Dylan Simon
 Louis Thiry
 Sergey Zagoruyko
-Jonas1312
\ No newline at end of file
+Jonas1312
```

Comment:
i would strongly prefer to stick to a first name - last name basis

Reply:
some people might not want to reveal their name..(i'm fine with it)

Reply:
it's trivial to find his name from his GitHub handle. this argument doesn't hold

Reply:
@Jonas1312 Do you have a preference? If not, it would probably be nice to change to first-last for consistency

Reply:
it's been a month since this. let's make the replacement to first name - last name

Reply:
OK can you do it?